File: Java/topflytechCodec/src/com/topflytech/codec/entities/PTT/VoiceMessage.java
package com.topflytech.codec.entities.PTT;
 
import com.topflytech.codec.entities.Message;

public class VoiceMessage  extends Message {
    private int encodeType;
    private byte[] voiceData;

    public int getEncodeType() {
        return encodeType;
    }

    public void setEncodeType(int encodeType) {
        this.encodeType = encodeType;
    }

    public byte[] getVoiceData() {
        return voiceData;
    }

    public void setVoiceData(byte[] voiceData) {
        this.voiceData = voiceData;
    }
}

File: TopflytechCodec/C#/TopflytechCodec/TopflytechCodec/Entities/PTT/VoiceMessage .cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace TopflytechCodec.Entities
{
    public class VoiceMessage : Message
    {
        private int encodeType;
        private byte[] voiceData;

        public int EncodeType
        {
            get { return encodeType; }
            set { encodeType = value; }
        }

        public byte[] VoiceData
        {
            get { return voiceData; }
            set { voiceData = value; }
        }
    }
}
